app/models/ConferenceDescriptor.scala

```diff
@@ -257,6 +257,11 @@ object ConferenceDescriptor {
     val NEUILLY_231_232 = Room("neu_232_232", "Neuilly 231M-232M", 32, "classroom", "rien", "231+232M", Some("Mezzanine South Neuilly"))
     val NEUILLY_234_235 = Room("neu_234_235", "Neuilly 234M-235M", 24, "classroom", "rien", "234+235M", Some("Mezzanine South Neuilly"))
 
+    val NEUILLY_LAB1 = Room("lab1", "Neuilly LAB 1", 30, "classroom", "rien", "LAB 1", Some("Neuilly"))
```

Comment:
```suggestion
    val NEUILLY_LAB1 = Room("lab1", "Neuilly LAB 1", 20, "classroom", "rien", "LAB 1", Some("Neuilly"))
```

@nicmarti maybe 20 as shared by @agoncal